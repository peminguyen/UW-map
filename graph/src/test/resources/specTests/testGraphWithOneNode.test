CreateGraph graphWithOneNode

AddNode graphWithOneNode A

ListNodes graphWithOneNode

ListChildren graphWithOneNode A