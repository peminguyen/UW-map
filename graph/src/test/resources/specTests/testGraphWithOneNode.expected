created graph graphWithOneNode

added node A to graphWithOneNode

graphWithOneNode contains: A

the children of A in graphWithOneNode are: